google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploymentConfigurator.java

```diff
@@ -52,7 +57,20 @@ public AppEngineDeploymentConfigurator(Project project) {
   @NotNull
   @Override
   public List<DeploymentSource> getAvailableDeploymentSources() {
-    List<DeploymentSource> deploymentSources = new ArrayList<DeploymentSource>();
+    List<DeploymentSource> deploymentSources = new ArrayList<>();
+
+    for (Module module : ModuleManager.getInstance(project).getModules()) {
+      if (isMavenBuild(module)) {
+        deploymentSources.add(
+            new MavenBuildDeploymentSource(
+                ModulePointerManager.getInstance(project).create(module), project));
+      }
+      if (isGradleBuild(module)) {
```

Comment:
Yes, in which case the user can choose between them. Not a common scenario, but as an example, I tested it with the spring boot sample project which has both gradle and maven configs.